kuma/wiki/views/list.py

```diff
@@ -8,9 +8,8 @@
 
 from ..constants import DOCUMENTS_PER_PAGE
 from ..decorators import process_document_path, prevent_indexing
-from ..models import (Document, DocumentTag, Revision, ReviewTag,
+from ..models import (Document, DocumentTag, ReviewTag,
                       LocalizationTag)
-from ..queries import MultiQuerySet
```

Comment:
This was the only user of MultiQuerySet. That code should be deleted if we end up not using it.